.travis.yml

```diff
@@ -14,17 +16,18 @@ cache:
   directories:
     - env/bin
     - env/lib/python2.7/site-packages
+    - node_modules
 install:
  - if [ "${TRAVIS_BRANCH}" = "master" -a "${TRAVIS_PULL_REQUEST}" = "false" ]; then rm -rf env; fi
  - touch requirements.txt package.json
- - make env
+ - make env -j2
  - env/bin/pip install --upgrade ijson==2.3.0
 before_script:
  - echo "DATABASE_URL=dbname=gratipay" | tee -a tests/local.env local.env
  - echo "REQUIRE_YAJL=true" | tee -a tests/local.env local.env
  - psql -U postgres -c 'CREATE DATABASE "gratipay";'
  - if [ "${TRAVIS_BRANCH}" = "master" -a "${TRAVIS_PULL_REQUEST}" = "false" ]; then rm -rfv tests/py/fixtures; fi
-script: LD_LIBRARY_PATH=/usr/local/lib make bgrun test doc
+script: LD_LIBRARY_PATH=/usr/local/lib make bgrun test doc -j2
```

Comment:
Since our Makefile is mostly phony targets I doubt this will make a difference, but from what I can tell it doesn't hurt. Did you actually benchmark?